egs/wsj/s5/steps/nnet3/compute_output.sh

```diff
@@ -0,0 +1,199 @@
+#!/bin/bash
+
+# Copyright 2012-2015  Johns Hopkins University (Author: Daniel Povey).
+#                2016  Vimal Manohar
+# Apache 2.0.
+
+# This script does forward propagation through a neural network. 
+# If the neural net was built on top of fMLLR transforms from a conventional
+# system, you should provide the --transform-dir option.
+
+# Begin configuration section.
+stage=1
+transform_dir=    # dir to find fMLLR transforms.
+nj=4 # number of jobs.  If --transform-dir set, must match that number!
+cmd=run.pl
+use_gpu=false
+frames_per_chunk=50
+iter=final
+extra_left_context=0
+extra_right_context=0
+extra_left_context_initial=-1
+extra_right_context_final=-1
+frame_subsampling_factor=1
+feat_type=
+compress=false    # Specifies whether the output should be compressed before
+                  # dumping to disk
+online_ivector_dir=
+priors=           # Priors vector to convert posteriors to pseudo-log
+                  # likelihoods
+output_name=      # Dump outputs for this output-node
+use_raw_nnet=true  # Use raw neural-network (without transition model)
+apply_exp=false  # Apply exp i.e. write likelihoods instead of log-likelihoods
+# End configuration section.
+
+echo "$0 $@"  # Print the command line for logging
+
+[ -f ./path.sh ] && . ./path.sh; # source the path.
+. parse_options.sh || exit 1;
+
+if [ $# -ne 3 ]; then
+  echo "Usage: $0 [options] <data-dir> <nnet-dir> <output-dir>"
+  echo "e.g.:   steps/nnet3/compute_output.sh --nj 8 \\"
+  echo "--online-ivector-dir exp/nnet3/ivectors_test_eval92 \\"
+  echo "    data/test_eval92_hires exp/nnet3/tdnn exp/nnet3/tdnn/output"
```

Comment:
nnet3-am-compute exists in the PR. The output is different if its a raw nnet vs not. One is log_likelihoods after subtracting log_priors, while the other is just the output.
The fMLLR input features still exist in decode.sh, which is where I copies most of the options from (extra-left-context etc.)